python/PIK_maya_loader/module/PIK_maya_loader.mod

```diff
@@ -0,0 +1 @@
++ MAYAVERSION:2025 PLATFORM:win64 PIK_maya_loader 0.0.1 ../
```

Comment:
The version must be also updated here. Maybe this can be updated by rez during build ? (using Jinja: https://jinja.palletsprojects.com/en/3.1.x/)